pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```
@@ -230,6 +230,9 @@ qtModule {
     [Paths]
     Prefix = ..
     EOF
+  '' + lib.optionalString (lib.versionAtLeast qtCompatVersion "5.15") ''
+    # Fix for out-of-sync QtWebEngine and Qt releases (since 5.15.3)
+    sed 's/5.15.3 /5.15.2 /' -i "$dev"/lib/cmake/*/*Config.cmake
```

```suggestion
    sed 's/${version} /${qtCompatVersion} /' -i "$dev"/lib/cmake/*/*Config.cmake
```
We should somehow pass the version to the module's expression. But I guess a comment where its version is defined is fine for now.